src/Main.java

```diff
@@ -1,5 +1,97 @@
+import java.util.Scanner;
+
 public class Main {
+
     public static void main(String[] args) {
 
+        Scanner input = new Scanner(System.in);
+        Pokemon playedCharacter = new Pokemon();
+        Pokemon challengedCharacter = new Pokemon();
+        Battle battle = new Battle();
+
+        Pokemon golduck = new WaterPokemon("Golduck", 12, 80, "pure", "Mega Launcher", "water pulse");
+        Pokemon vulcanion = new FirePokemon("Vulcanion", 18, 62, " Primary Fire-type", "water-type", "magic", "Blue Flare");
+        Pokemon bulbasaur = new GrassPokemon("Bulbasur", 9, 42, "Grass Knot", "poison powder", "Magical Leaf");
+        Pokemon manectric = new ElectricPokemon("Manectric", 8, 32, "Spark", "Zing Zap");
+
+        // een te spelen karakter kiezen:
+        System.out.print("Welk character wil je spelen? \n1. Vulcanion\n2. Golduck\n3. Bulbasaur\n4. Manectric\n" +
+                "jou keuze is: ");
+        int characterChoice = input.nextInt();
+        System.out.println();
+
+        switch (characterChoice) {
+            case 1:
+                System.out.println(vulcanion.toString());
+                playedCharacter = vulcanion;
+                break;
+            case 2:
+                System.out.println(golduck.toString());
+                playedCharacter = golduck;
+                break;
+            case 3:
+                System.out.println(bulbasaur.toString());
+                playedCharacter = bulbasaur;
+                break;
+            case 4:
+                System.out.println(manectric.toString());
+                playedCharacter = manectric;
+                break;
+            default:
+                // vraag: met welke commando zou ik het programma helemaal kunnen stoppen?
+                System.out.println("ongeldige keuze");
+                break;
+        }
+
+        // vraag: of speler HP punten wil weten van gekozen karakter.
+        System.out.println();
+        System.out.print("Wil je mijn HP punten weten ? (j/n) ");
+        String hpRequest = input.next();
```

Comment:
Wees je wel bewust van het verschil tussen next() en nextLine(). https://www.geeksforgeeks.org/difference-between-next-and-nextline-methods-in-java/